src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
```

Comment:
You may want to validate the 'user' object before using it in your component to avoid possible runtime errors if the object or its properties are undefined.